packages/main/src/themes/Label.css

```diff
@@ -1,38 +1,58 @@
 :host(:not([hidden])) {
 	display: inline-flex;
 }
-
 :host {
 	max-width: 100%;
 	color: var(--sapUiContentLabelColor);
 	font-family: var(--sapUiFontFamily);
 	font-size: var(--sapMFontMediumSize);
 	font-weight: normal;
 	cursor: text;
+}	
+:host(:not([wrap])) .ui5-label-root {
+	font-weight: normal;
```

Comment:
We are setting font-weight: normal; on the :host only, beacuse this way the user can do the following and it will be applied.

```css
ui5-label {
     font-weight: bold;
}
```
Now, this would not take effect. Instead try font-weight: inherit